doc/components.md

```diff
@@ -492,6 +492,21 @@ Check the example app
 [nine_tile_box](https://github.com/flame-engine/flame/blob/main/examples/lib/stories/utils/nine_tile_box.dart)
 for details on how to use it.
 
+## CustomPainterComponent
+
+A `CustomPainter` is a useful Flutter class used with the `CustomPaint` widget to render custom
+shapes inside a Flutter application.
+
+Flame provides a component that can render a `CustomPainter` called `CustomPainterComponent`, it
+receives a custom painter and just render it on the game canvas.
```

Comment:
```suggestion
receives a custom painter and just renders it on the game canvas.
```

Reply:
I would say that the word `just` can also be removed, it adds no extra information and breaks up the sentence unnecessarily imho.

Reply:
Ageed, edited the suggestion.